src/js/renderers/editor-dom.js

```diff
@@ -43,6 +44,9 @@ function renderHTMLText(marker) {
   text = text.replace(/ ( )/g, () => {
     return ' '+NO_BREAK_SPACE;
   });
+  text = text.replace(/\t/g, () => {
```

Comment:
The replace function argument is not necessary, also. Not sure if there's any performance difference, but this code is equivalent and (imo) a little more understandable: `text = text.replace(/\t/g, TAB_CHARACTER);`